hip_data_tools/apache/cassandra.py

```diff
@@ -160,6 +160,7 @@ class CassandraConnectionSettings:
     load_balancing_policy: LoadBalancingPolicy
     secrets_manager: CassandraSecretsManager
     ssl_options: dict = None
+    consistency_level: Optional[ConsistencyLevel] = ConsistencyLevel.LOCAL_ONE
```

Comment:
```suggestion
    consistency_level: Optional[ConsistencyLevel] = ConsistencyLevel.LOCAL_ONE
```
This may not work with legacy mode

Reply:
If we are assigning a default value to it, it doesn't need to be an `Optional` right?

Reply:
@Rukesh-Kapuluru  what is "legacy mode"?

Reply:
> If we are assigning a default value to it, it doesn't need to be an `Optional` right?

Good catch. Removed Optional from type hint.